tests/cli/test_rasa_data.py

```diff
@@ -152,6 +154,351 @@ async def mock_from_importer(importer: TrainingDataImporter) -> Validator:
         data.validate_files(args)
 
 
+def test_rasa_data_augment_nlu_intent_collection(run: Callable[..., RunResult],):
+    from rasa.nlu.data_augmentation.augmenter import (
+        _collect_intents_for_data_augmentation,
+    )
```

Comment:
To be consistent with the location of tests for their respective modules, I think all the tests that are testing smaller parts of the whole command like intents collection, paraphrase pool creation, etc. should go into a separate test file - `tests/nlu/data_augmentation/test_augmenter.py`